deploy/storage/bigtable.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
nit: Should we call this bigtable_import_groups.json or something to distinguish it from the current one?